src/Tribe/Aggregator.php

```diff
@@ -3,6 +3,14 @@
 defined( 'WPINC' ) or die;
 
 class Tribe__Events__Aggregator {
+	/**
+	 * Cache key used to storage the services list returned by the call to:
+	 * - Tribe__Events__Aggregator__Service::instance()->get_origins();
+	 *
+	 * @since TBD
+	 */
+	const SERVICES_CACHE_KEY = 'tribe_aggregator_services_list';
```

Comment:
Avoid new Constants, use a public prop. Access by `tribe( 'tec.aggregator' )->key_cache_services`.
When saving keys let's try to always start with `key_...`

Reply:
Curios here of the reasons behind this also wondering if this should be the approach from now on as in some places the `CONST` is the current way to do this.

Reply:
Yes, so the problem is that there is no way to deprecate `CONST` on PHP 5.2 so we are avoiding adding any new ones. I have a ticket out for me to update that on our Engineering Docs.